frontend/src/main/pages/LeaderboardPage.js

```diff
@@ -46,6 +48,8 @@ export default function LeaderboardPage() {
     );
   // Stryker restore all 
 
+  const navigate = useNavigate();
+
   const showLeaderboard = (hasRole(currentUser, "ROLE_ADMIN") || commons.showLeaderboard );
   return (
     <div data-testid={"LeaderboardPage-main-div"} style={{backgroundSize: 'cover', backgroundImage: `url(${Background})`}}>
```

Comment:
My only comment is that maybe we can put the back button at the top of the page so that users don't have to scroll all the way down for it. Something like:

```js
<h1>Leaderboard</h1>
<Button style="text-align: right;" onClick={() => navigate(-1)} data-testid="LeaderboardPage-back-button" >
                    Back
</Button>
...
```
Just my suggestion. I'm also fine with how it is currently implemented.